cmake/Scripts/configure

```diff
@@ -48,6 +48,7 @@ Optional Packages:
   --with-superlu=PATH     user defined path to SuperLU library
   --with-umfpack=PATH     use UMFPACK/SuiteSparse from a specified location
   --with-ert=PATH         Use ERT libraries
+  --with-zoltan=PATH      Use ZOLTAN libraries
```

Comment:
It is perhaps a little unfair to require this now, since it wasn't there already for the other options, but it made me realise I had some questions:
 - Which path *is* required, is it the "prefix" path (e.g. /usr/local) or the full path (e.g. /usr/local/lib)?
 - Is this behaviour uniform across the PATH arguments?